gui/default/assets/css/overrides.css

```diff
@@ -556,3 +564,12 @@ html[lang|="ko"] i {
     -webkit-user-select: all;
     user-select: all;
 }
+
+/* Align checkbox with button in login forms */
+#login-options .stay-logged-in {
+  margin-top: 0.25em;
+}
```

Comment:
```suggestion
```

I thought we'd got rid of this?

Reply:
Thanks, my bad - looks like I deleted this only in speculative commits that ultimately didn't make it into https://github.com/emlun/syncthing/pull/8 .

Reply:
Fixed in 0905f07273ac69f5ac33ea689ea57a309a03844b.